### PR TITLE
Creates a Docs site

### PR DESCRIPTION
**Pull request by olirice**
## What kind of change does this PR introduce?
Adds a docs site https://supabase-community.github.io/vec2pg/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,23 +18,25 @@
...
 ---
 
+**Documentation**: <a href="https://supabase-community.github.io/vec2pg" target="_blank">https://supabase-community.github.io/vec2pg</a>
+
 **Source Code**: <a href="https://github.com/supabase-community/vec2pg" target="_blank">https://github.com/supabase-community/vec2pg</a>
 
 ---
 
-A CLI for migrating data from vector databases to [Supabase](https://supabase.com).
+`vec2pg` is a CLI for migrating data from third party vector databases to [Supabase](https://supabase.com) with pvector.
```
Comment (juancarlospaco) [inline README.md]:
```suggestion
`vec2pg` is a CLI tool for migrating data from third-party vector databases to [Supabase](https://supabase.com) with Pgvector.
```

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Contributing
+
+`vec2pg` is open source software. External contributions are welcome. Note that we have a high bar for testing.
+
+Before opening a PR, please [create an issue](https://github.com/supabase-community/vec2pg/issues/new/choose) in GitHub to discsus and approve the change you're interested in making.
```
Comment (juancarlospaco) [inline docs/contributing.md]:
```suggestion
Before opening a PR, please [create an issue](https://github.com/supabase-community/vec2pg/issues/new/choose) in GitHub to discuss and approve the change you're interested in making.
```

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+---
+
+**Documentation**: <a href="https://supabase-community.github.io/vec2pg" target="_blank">https://supabase-community.github.io/vec2pg</a>
+
+**Source Code**: <a href="https://github.com/supabase-community/vec2pg" target="_blank">https://github.com/supabase-community/vec2pg</a>
+
+---
+
+`vec2pg` is a CLI for migrating data from third party vector databases to [Supabase](https://supabase.com).
```
Comment (juancarlospaco) [inline docs/index.md]:
```suggestion
`vec2pg` is a CLI tool for migrating data from third-party vector databases to [Supabase](https://supabase.com).
```

**Comment by olirice** [comment]
thanks @juancarlospaco

**Pull request merged**